include/bluetooth/services/hids_c.h

```diff
@@ -220,340 +189,364 @@ struct bt_gatt_hids_c_info_val {
  *
- * @note This function clears whole memory occupied by the object.
+ * @note This function clears the whole memory occupied by the object.
  *       Do not call it on the object in use.
  *
- * @param hids_c HIDS client object
+ * @param hids_c HIDS client object.
  * @param params Initialization parameters.
  */
 void bt_gatt_hids_c_init(struct bt_gatt_hids_c *hids_c,
 			const struct bt_gatt_hids_c_init_params *params);
 
 /**
- * @brief Assign handlers to the HIDS client instance
+ * @brief Assign handlers to the HIDS client instance.
  *
- * This function should be called when a link with a peer has been established
+ * This function should be called when a link with a peer has been established,
  * to associate the link to this instance of the module. This makes it
  * possible to handle several links and associate each link to a particular
  * instance of this module. The GATT attribute handles are provided by the
- * GATT DB discovery module.
+ * GATT Discovery Manager.
  *
  * @note
- * This function would start report discovery process.
+ * This function starts the report discovery process.
  * Wait for one of the functions in @ref bt_gatt_hids_c_init_params
- * before using HIDS object.
+ * before using the HIDS object.
  *
  * @param dm     Discovery object.
  * @param hids_c HIDS client object.
  *
  * @retval 0 If the operation was successful.
- * @retval (-ENOTSUP) Special error code used when UUID
+ *           Otherwise, a (negative) error code is returned.
+ * @retval (-ENOTSUP) Special error code used when the UUID
  *         of the service does not match the expected UUID.
- * @retval Otherwise, a negative error code is returned.
  */
 int bt_gatt_hids_c_handles_assign(struct bt_gatt_dm *dm,
 				  struct bt_gatt_hids_c *hids_c);
 
 /**
- * @brief Release the HIDS client instance
+ * @brief Release the HIDS client instance.
  *
- * Function releases all the dynamic memory allocated by the HIDS client
+ * This function releases all dynamic memory allocated by the HIDS client
  * instance and clears all the instance data.
  *
  * @note
- * Do not call this function on instance that was not initialized
+ * Do not call this function on an instance that was not initialized
  * (see @ref bt_gatt_hids_c_init).
- * After the instance is initialized it is safe to call this function
+ * After the instance is initialized, it is safe to call this function
  * multiple times.
  *
- * @param hids_c The HIDS client object.
+ * @param hids_c HIDS client object.
  */
 void bt_gatt_hids_c_release(struct bt_gatt_hids_c *hids_c);
 
 /**
- * @brief Abort all pending transfers
+ * @brief Abort all pending transfers.
  *
- * Call this function @b when server was disconnected.
- * In other case if there is any pending read or write transfer
- * it might permanently return EBUSY error on next transfer.
+ * Call this function only when the server is disconnected.
+ * In other case, if there were any read or write transfers pending,
```

Comment:
Change to:
Otherwise, if any read or write transfers are pending, the HIDS client object might permanently return an EBUSY error on the following transfer attempts.

Reply:
Done